### PR TITLE
Bugfix/increase state budget

### PR DESCRIPTION
**Pull request by liayoo**
- Increase STATE_TREE_BYTES_LIMIT and --max-old-space-size to 5GB

### CODE REVIEW

File: start_node_incremental_gcp.sh [hunk 1]
```diff
@@ -147,7 +147,7 @@ eval $KILL_CMD
 printf "\n#### [Step 6] Start new node server ####\n\n"
 
 sleep 10
-MAX_OLD_SPACE_SIZE_MB=4000
+MAX_OLD_SPACE_SIZE_MB=5000
```
Comment (platfowner) [inline start_node_incremental_gcp.sh:150]: Taking overhead into account, could you please use 6000 instead?

File: start_node_genesis_gcp.sh [hunk 2]
```diff
@@ -109,7 +109,7 @@ export STAKE=100000
 export BLOCKCHAIN_DATA_DIR="/home/ain_blockchain_data"
 
 printf "\nStarting up Blockchain Node server..\n\n"
-START_CMD='nohup node --async-stack-traces --max-old-space-size=4000 client/index.js >/dev/null 2>error_logs.txt &'
+START_CMD='nohup node --async-stack-traces --max-old-space-size=5000 client/index.js >/dev/null 2>error_logs.txt &'
```
Comment (platfowner) [inline start_node_genesis_gcp.sh:112]: ditto
and could you define variable MAX_OLD_SPACE_SIZE_MB here too?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks for handling this issue!

File: start_node_incremental_gcp.sh (hunk 1 above)
Comment (liayoo) [reply start_node_incremental_gcp.sh]: Done!

File: start_node_genesis_gcp.sh (hunk 2 above)
Comment (liayoo) [reply start_node_genesis_gcp.sh]: Done!

**Pull request merged**